docs/nightly/en/user-guide/operations/disaster-recovery/back-up-&-restore-data.md

```diff
@@ -1,47 +1,96 @@
-# Back up & restore data
+# Back up and Restore Data
 
-Use [`COPY` command](/reference/sql/copy.md)to backup and restore data.
+It is crucial to back up your databases to ensure that you can recover your data and quickly resume operations in the event of problems such as system crashes, hardware failures, or accidental data deletion by users.
 
-## Backup Table
+This document provides instructions on how to back up and restore data using the [Greptime command line](/reference/command-lines.md) and the SQL [`COPY` command](/reference/sql/copy.md).
 
-Backup the table `monitor` in `parquet` format to the file `/home/backup/monitor/monitor.parquet`:
+## Back up and restore schemas
+
+Before backing up and restoring data for tables or databases,
+it is necessary to back up and restore the schemas.
+
+### Back up schemas
+
+The following example command line connects to the GreptimeDB server at `127.0.0.1:4000` and exports the `CREATE TABLE` SQL statements to the folder `/home/backup/schema/`:
+
+```shell
+greptime cli export --addr '127.0.0.1:4000' --output-dir /home/backup/schema/ --target create-table
+```
+
+### Restore schemas
+
+To restore the schema to a specified database, use the PostgreSQL client.
+For example,
+the following command line runs the `CREATE TABLE` SQL statements in the file `greptime-public.sql` and creates table to the `public` database:
```

Comment:
Should we highlight that user must use `PostgreSQL` client or via Http API to execute the commands?

Reply:
we mentioned above using the PG Client and the sample shell also uses the PG command line. Maybe that's enough.